### DISCUSSION TITLE
What happen with playlist system?

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#14201] [Q&A] [answered]
It was in multi a while ago, but now I can only choose one map.

**Comment by frenzibyte** [comment, marked as answer]
Can you please begin by describing specifically what your issue is? and/or attach a video of it. Also please attach the logs.

**Reply by Moltanicaa** [reply]
This is not issue, this is question.

**Reply by frenzibyte** [reply]
I still don't understand what you're asking? playlists existing in multiplayer? that never happened before, but it's something that'll be supported eventually.

I'm presuming you're mistaking the "multi from a while ago" being "timeshift" which has now been renamed to "playlists" next to the known real-time multiplayer.

**Reply by Moltanicaa** [reply]
Yes, that's it, thanks. Now it is clear.
